tests/browser/html.js

```diff
@@ -0,0 +1,8 @@
+var doc = typeof window !== 'undefined'
+  ? document
+  : new (require('jsdom').JSDOM)().window.document
```

Comment:
since this may be bundled by browserify for the transform and babel plugin tests, and we don't need jsdom in the browser because there's a real DOM, this trick can be used to avoid bundling it:
```suggestion
  : new (require('js' + 'dom').JSDOM)().window.document
```
because browserify doesn't process require calls with a "dynamic" argument, only plain strings.